### ISSUE TITLE
[Bug] Changed usernames don't update until you refresh

### ISSUE THREAD

**Issue by Not-Kris** [dubtrack/www-dubtrack-fm#161]
After someone changes their username, if you click on their name in the userlist and press @ to mention them, it will mention the old name. Their profile popup also shows the old name.
![image](https://cloud.githubusercontent.com/assets/12618255/11069002/64afea54-87d4-11e5-84f3-623183076363.png)

**Comment by tris90190** [comment]
I agree Coding to the project will be fixed soon. Thanks for The Request!

**Comment by Larry1123** [comment]
I think this is fixed. Would like to know if it still happens.

**Comment by KRDemoz** [comment]
Confirmed still happening.

Name change shows on chat but on the popup it remains the old name.
Clicking on the name to @ mention them will show the new name, only the popup is the issue.
